### PR TITLE
How to use guide

### PR DESCRIPTION
**Pull request by johannest**
How to use guide

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,3 +99,50 @@ public class Example
...
+5. Run `java declarative.DesignToJavaConverter -b base.package.name -s path/to/declarative/file/MyViewDesign.html`
+
+6. Take console output and create a new Java (e.g. `MyViewJava.java`) class corresponding your `MyViewDesign.java` class
+
+7. Clean up the Java class (`MyViewJava.java`)
+	- fix possible autowired thing
+	- check the warnings (e.g. "Warning! Unsupported class...") from console output and fix in the code if needed
+	- make sure both `MyViewJava.java` and `MyViewDesign.java` extend same (layout) class
+	- refactor the content
+	- fix numeric literals
```
Comment (elmot) [inline README.md:144]: What is wrong with numeric literals? What should be fixed?

**Review by elmot — COMMENTED** [review]
looks like you have tested the converter against Vaadin 8. 
If it works, we can mention that in the very first line of README.

File: README.md (hunk 1 above)
Comment (johannest) [reply README.md:144]: For example `horizontalLayout1.setExpandRatio(newCommentInput, 1.0D);` D have to be converted fo f

Comment (johannest) [reply README.md:144]: And yes, it seems to work also with Vaadin 8

Comment (elmot) [reply README.md:144]: added as an issue.

**Review by elmot — APPROVED** [review]

**Pull request merged**